src/quest_manager/templates/quest_manager/submission_buttons_form_student.html

```diff
@@ -0,0 +1,17 @@
+<div class="btn-group" role="group">
+
+    {% if not completed %}
+        <button type='submit' name='complete' class='btn btn-info'
+                title='RE-SUBMIT this quest for completion'>
```

Comment:
"...quest for completion"  This is awkward phrasing.  I know you got it from the other button, which is also awkard!  Can you change both of them (this and the old one) to "Submit/Re-submit this quest for approval" That makes way more sense.

Reply:
"for completion" on buttons and in project documentation changed to "for approval" where appropriate.